docs/version0.46.1.md

```diff
@@ -0,0 +1,47 @@
+<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# What's new in version 0.46.1
+
+The following new features and notable changes since version 0.46.0 are included in this release:
+
+- [New binaries and changes to supported environments](#binaries-and-supported-environments)
+
+
+## Features and changes
+
+### Binaries and supported environments
+
+Eclipse OpenJ9&trade; release 0.46.1 supports OpenJDK 8, 11, 17, 21, and 22.
+
+Support for running OpenJ9 with all versions of OpenJDK on CentOS 6, CentOS 7, Red Hat Enterprise Linux (RHEL) 6, and RHEL 7 is removed.
```

Comment:
I know this is a copy of the statement in 0.46, but I wonder if it can be worded in a different way. We didn't change the JVM or how it's built, but we are no longer claiming these OSes are supported platforms.

Reply:
Can we delete this line then because the support was already removed in 0.46.0 release? Or is there any other information that is required here?

Reply:
Yes, it can be deleted since it isn't a change from 0.46

Reply:
Although it would be nice to update the 0.46 topic with new wording, to be clear about what "support is removed" means.

Reply:
Is the following content applicable for OpenJ9?

No release binaries or packages will be produced for OpenJ9 on these platforms and no support will be provided (including interim fixes in support cases).

Reply:
Technically OpenJ9 is a source code only project. These things refer to Semeru, which is a down stream product that uses OpenJ9.

Although we aren't testing OpenJ9 on these out of public support platforms, it can still be built and run there, there were no code changes to prevent it. Any request for OpenJ9 open source support for an OS specific problem on these OSes will be refused, but for a bug in OpenJ9 unrelated to the OS, it doesn't really matter on what OS it was found.

Reply:
What I don't like about the existing wording is that it may imply we changed something in OpenJ9. What I'm looking for is something like:

CentOS 6, CentOS 7, Red Hat Enterprise Linux (RHEL) 6, and RHEL 7 are removed from the list of supported platforms.